libs/optimizer/src/lib/optimizer.ts

```diff
@@ -0,0 +1,348 @@
+import { Solution, solver } from 'igc-xc-score';
+import { BRecord, IGCFile } from 'igc-parser';
+import { createSegments } from './utils/createSegments';
+import { concatTracks } from './utils/concatTracks';
+import { ScoringRules, scoringRules } from './scoringRules';
+import { getDistance } from 'geolib';
+
+// When the track has not enough points (<5), we build a new one by adding interpolated points between existing ones.
+// see this issue https://github.com/mmomtchev/igc-xc-score/issues/231 for removing this limitation
```

Comment:
nit: you can drop "for removing this limitation"

Reply:
Ok. Changed